tst/test_config_space.py

```diff
@@ -132,6 +134,19 @@ def test_serialization():
         assert dct1 == dct2
 
 
+def test_config_space_from_and_to_dict():
+    config_space = {
+        "int": randint(1, 2),
+        "logint": lograndint(3, 4),
+        "float": uniform(5.5, 6.5),
+        "logfloat": loguniform(7.5, 8.5),
+        "categorical": choice(["a", "b", "c"]),
+        "normal": randn(2.0, 1.0),
+        "const_str": "constant",
```

Comment:
We would probably want to add an integer and a float to cover all the union types.

Reply:
added both cases